darwin/exporter/formats/yolo.py

```diff
@@ -31,7 +31,7 @@ def export(annotation_files: Iterable[dt.AnnotationFile], output_dir: Path) -> N
 
 def _export_file(annotation_file: dt.AnnotationFile, class_index: ClassIndex, output_dir: Path) -> None:
     txt = _build_txt(annotation_file, class_index)
-    output_file_path = (output_dir / annotation_file.filename).with_suffix(".txt")
+    output_file_path = (output_dir / annotation_file.path.stem).with_suffix(".txt")
```

Comment:
There was an issue (not here I don't think) re using stem. I believe it was for filenames that included '.' character, 
`one.two.jpeg` for instance the .stem function loses that middle section IIRC becoming `one.jpeg`.  This did crop up as an issue elsewhere (I think for Brent, but I don't recall the ticket). Might want to verify that double dotted filenames also work.

Reply:
Well remembered, that was [IO-1081](https://linear.app/v7labs/issue/IO-1081/bug-pulling-a-release-with-cli-on-darwin-py-0822-and-0823-truncates)

Could we just apply that fix here? Instead of using `stem.with_suffix`, just `(stem + suffix)`